SixTrack/scatter.s90

```diff
@@ -181,6 +182,8 @@ end function
 subroutine scatter_profile_getParticle(profileIdx, x,y, xp,yp,E)
 
     implicit none
+
+    ! Input/Output
```

Comment:
I would rather call that arguments, or to be technically correct, dummy variables.

I/O implies something being sent out of the "computer", i.e. to disk, terminal, network, etc. ...